packages/desktop-client/src/components/accounts/Account.tsx

```
@@ -702,6 +702,58 @@
+      );
+      const changedTransactions = [];
+      for (const transaction of transactions) {
+        const res: TransactionEntity | null = await send('rules-run', {
+          transaction,
+        });
+        if (res) {
+          changedTransactions.push(res);
+        }
+      }
+      // Bulk fetch transactions
+      const transactions = this.state.transactions.filter(trans =>
```

Biome (1.9.4)

[error] Shouldn't redeclare 'transactions'. Consider to delete it or rename it.

'transactions' is defined here:


(lint/suspicious/noRedeclare)